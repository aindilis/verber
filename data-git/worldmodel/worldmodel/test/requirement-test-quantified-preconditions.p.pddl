(define
 (problem REQUIREMENT-TEST-QUANTIFIED-PRECONDITIONS)
 (:domain REQUIREMENT-TEST-QUANTIFIED-PRECONDITIONS)
 (:objects obj2 obj1 - type1)
 (:init
  (pred1 obj1))
 (:goal
  (and
   (pred1 obj2))))