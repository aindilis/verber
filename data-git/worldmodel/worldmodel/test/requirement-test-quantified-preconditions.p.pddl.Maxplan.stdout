MaxPlanII beta.

bb: parsing domain file
domain 'REQUIREMENT-TEST-QUANTIFIED-PRECONDITIONS' defined
 ... done.
bb: parsing problem file
problem 'REQUIREMENT-TEST-QUANTIFIED-PRECONDITIONS' defined
 ... done.


******  Solve with 1 layers Start******

creating action-based encoding:
        building rplan graph layers, up to layer [1];
        goal constraints;
        building exclusion constraints --> up to layer 1...

DECISION LAYER 1, WRITING ACTION-BASED ENCODING WITH 1 VARS, 1 CLAUSES.
1 G clauses, 0 A clauses, 0 E clauses.c CNF file planning task -p , -o requirement-test-quantified-preconditions.d.pddl, -f requirement-test-quantified-preconditions.p.pddl, bound 1, action based encoding
In building up solver: p cnf 1 1
==================================[ MINISAT ]===================================
| Conflicts |     ORIGINAL     |              LEARNT              | Progress |
|           | Clauses Literals |   Limit Clauses Literals  Lit/Cl |          |
==============================================================================
|         0 |       0        0 |       0       0        0     0.0 |  0.000 % |
==============================================================================
Totoal time used by londex constraint checking is [B0.000000][L0.000000],apply counts 1, apply_generated 0
restarts          :            1
conflicts         :            0           (      nan / sec      )
decisions         :            1           (      inf / sec      )
propagations      :            1           (      inf / sec      )
inspects          :            0           (      nan / sec      )
conflict literals :            0           (      nan % deleted  )
CPU time          :         0.00 sec
SATISFIABLE

Info:      0.01 seconds total solving time
           0.00 MB memory used
        solution found at layer 1

